File: Resource/51900286-51900712-51900718-51900699/source demo/Web Application/ComNha_API/ComNha_API/Models/Banan.cs
using System;
using System.Collections.Generic;

namespace ComNha_API.Models
{
    public partial class Banan
    {
        public Banan()
        {
            Lichhens = new HashSet<Lichhen>();
            Phieugoimons = new HashSet<Phieugoimon>();
        }

        public string MaBanAn { get; set; } = null!;
        public string TinhTrang { get; set; } = null!;
        public int SucChua { get; set; }
        public string GhiChu { get; set; } = null!;

        public virtual ICollection<Lichhen> Lichhens { get; set; }
        public virtual ICollection<Phieugoimon> Phieugoimons { get; set; }
    }
}

File: Dev/51900718/src/main/java/tdtu/advanced/java/thinh68/models/BanAn.java
package tdtu.advanced.java.thinh68.models;

import java.util.List;
import java.util.Set;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
 
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Entity
@Table(name = "ban_an")
public  class BanAn {
    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    @Column(name="ma_ban_an")
    private Long maBanAn;
    
    @Column(name="tinh_trang", nullable = false)
    private String tinhTrang;
    	
    @Column(name="suc_chua", nullable = false)
    private Integer sucChua;
    
    @Column(name="ghi_chu")
    private String ghiChu;
    
    @OneToMany(mappedBy = "banAn")
    private List<LichHen> banAn_LichHens;

    @OneToMany(mappedBy = "banAn")
    private List<PhieuGoiMon> banAn_PhieuGoiMons;
}